vizro-core/src/vizro/static/css/dropdown.css

```diff
@@ -131,11 +138,12 @@ div.nav_panel a {
 
 /** spacing for tag elements only when elements are present inside
 wrapper **/
+/* Padding allows elements not to jump but may be too large */
 .Select--multi.has-value .Select-multi-value-wrapper {
   display: flex;
   flex-wrap: wrap;
   gap: 4px;
-  padding: 4px 8px;
+  padding: 13px 8px;
```

Comment:
@nadijagraca Can you help me in fixing this behaviour: the current setting stops the jumping of the dropdown, but now the padding is a little too large for my taste. Have not been able to figure out why